help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,74 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports authentication using:
 
-To accommodate any enterprise needs, we can develop a customer-specific integration.
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
-All methods can be enabled separately or combined.
+You can enable all authentication methods separately or combined. To accommodate any enterprise needs, we can develop a customer-specific integration. 
 
 [Learn more about authentification](../govern/authentication.md).
 
 ## Common options
 
-If you want to display something on the login window, set the 'Prompt' option
-![Login prompt](login-prompt.png)
+To display something on the login window, set the **Prompt** option
+<p align="center">
+  <img src="login-prompt.png" alt="Login prompt" width="300"></img>
+</p>
 
-Enable the 'Send Welcome Email' option to send welcome emails to new signed-up users.
+Enable the **Send Welcome Email** option to send welcome emails to new signed-up users.
 
 ## Login-password authentication
 
-It is the most basic method to authenticate users with Datagrok.
-
 To configure login-password authentication for Datagrok:
 
-1. Go to the Datagrok Settings section 'Users and Sessions'; this section contains all authentication settings.
-2. Enable 'Internal authentication' to use the login-password method
-3. Signup option is alterable; you can disable it using the 'Signup Allowed' option
-4. To restrict from which domains people can sign up to the platform, use the 'Signup Domains Whitelist' option. You can
+1. Go to the Datagrok Settings section **Users and Sessions**; this section contains all authentication settings.
+2. Enable **Internal authentication** to use the login-password method
+3. Signup option is alterable; you can disable it using the **Signup Allowed** option
+4. To restrict from which domains people can sign up to the platform, use the **Signup Domains Whitelist** option. You can
    set several domains separated with commas
-5. Enable the 'Require Email Confirm' option to force people to use working emails.
-
-### Email service
+5. Enable the **Require Email Confirm** option to force people to use working emails.
 
-For login-password authentication, it is important to configure an Email service that will deliver signup, welcome,
+To use login-password authentication, it is important to configure an [Email service](configure-smtp.md) that will deliver signup, welcome,
```

Comment:
Are you sure 'Email service' should be capitalized?

Reply:
Let's make it remove some options; I think It can be something like
`that delivers signup and password emails.`